apps/ejabberd/src/ejabberd_auth_odbc.erl

```diff
@@ -333,24 +333,24 @@ scram_passwords(Server, ScramIterationCount) ->
 scram_passwords(Server, Count, Interval, ScramIterationCount) ->
     LServer = jid:nameprep(Server),
     ?INFO_MSG("Converting the stored passwords into SCRAM bits", []),
-    ToConvertCount = case catch odbc_queries:get_users_without_scram_count(LServer) of
-        {selected, [_], [{Res}]} -> binary_to_integer(Res);
+    ToConvertCount = case catch rdbms_queries:get_users_without_scram_count(LServer) of
+        {selected, [{Res}]} -> binary_to_integer(Res);
         _ -> 0
     end,
 
     ?INFO_MSG("Users to scrammify: ~p", [ToConvertCount]),
     scram_passwords1(LServer, Count, Interval, ScramIterationCount).
 
 scram_passwords1(LServer, Count, Interval, ScramIterationCount) ->
-    case odbc_queries:get_users_without_scram(LServer, Count) of
-        {selected, _, []} ->
+    case rdbms_queries:get_users_without_scram(LServer, Count) of
+        {selected, []} ->
             ?INFO_MSG("All users scrammed.", []);
-        {selected, [<<"username">>, <<"password">>], Results} ->
+        {selected, Results} ->
             ?INFO_MSG("Scramming ~p users...", [length(Results)]),
             lists:foreach(
               fun({Username, Password}) ->
                       Scrammed = prepare_scrammed_password(ScramIterationCount, Password),
-                      case catch odbc_queries:set_password_t(LServer, Username, Scrammed) of
+                      case catch rdbms_queries:set_password_t(LServer, Username, Scrammed) of
                           {atomic, ok} -> ok;
                           Other -> ?ERROR_MSG("Could not scrammify user ~s@~s because: ~p", [Username, LServer, Other])
```

Comment:
According to **Elvis**:
> Line 355 is too long:                           Other -> ?ERROR_MSG("Could not scrammify user ~s@~s because: ~p", [Username, LServer, Other]).